### PR TITLE
Release3.2

### PR DESCRIPTION
**Pull request by fepitre**
Add support for CentOS template creation

### CODE REVIEW

File: qubes-rpc/qfile-unpacker.c [hunk 1]
```diff
@@ -65,6 +65,7 @@ int main(int argc __attribute((__unused__)), char ** argv __attribute__((__unuse
 	switch (pid=fork()) {
 		case -1:
 			gui_fatal("Failed to create new process");
+			__attribute__ ((fallthrough));
```
Comment (marmarek) [inline qubes-rpc/qfile-unpacker.c]: Better mark gui_fatal with `noreturn`. Or add (never reachable) `exit(1)` here. Otherwise this will be misleading.

Comment (marmarek) [reply qubes-rpc/qfile-unpacker.c]: And also this breaks the build on older gcc (see travis linked below):
```
qfile-unpacker.c: In function 'main':
qfile-unpacker.c:68:4: error: empty declaration [-Werror]
    __attribute__ ((fallthrough));
    ^
cc1: all warnings being treated as errors
<builtin>: recipe for target 'qfile-unpacker.o' failed
```

**Comment by marmarek** [comment]
Also, as a policy, changes should go to master branch first, then be backported to stable release branch (if applicable).

**Comment by fepitre** [comment]
I will clean up things and pull request on the master branch

**Pull request closed without merging**